core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -0,0 +1,30 @@
+---
+defaultCluster: api.ci
+distribution:
+  api.ci:
+    jobs:
+    - apici-job1
+    - apici-job2
+    files:
+    - ".*-postsubmits.yaml$"
+    - ".*openshift/release/.*-periodics.yaml$"
+    - ".*-periodics.yaml$"
+  build01:
+    jobs:
+    - build01-job1
+    - build01-job2
+    - build01-job3
+  app.ci:
+    jobs:
+    - appci-job1
+volumeGroupIndex: -1 # -1 means no group has been migrated
```

Comment:
What for is this?

Reply:
This is tricky.
We have calculated jobs for a week with the number of each job occurring within that week.
Then we want to migrate jobs by the same volumes (1/5 of sum of job X number) each day.

We have that logic in the python script. I think we still need it here.

Reply:
Err so can we:
* Give a name to the groups rather than referencing them by index
* Add a `groups` parameter to the clusters and just put the group names there?

The current config scheme here is IMHO super unintuitive. Also I think how we constructed the group (By using 1/nth of the total job volume) shouldn't define how we name the config parameter.

Reply:
On it.

Reply:
Done. See if I do it right.

Reply:
lgmt. @petr-muller WDYT?